code/aepcomposeui/src/main/java/com/adobe/marketing/mobile/aepcomposeui/aepui/state/AEPUIState.kt

```diff
@@ -0,0 +1,6 @@
+package com.adobe.marketing.mobile.aepcomposeui.aepui.state
+
+/**
+ * Sealed interface representing different states of an AEP UI.
```

Comment:
What does `different states of an AEP UI` mean? An implementation of AEP UI can have only a single state. Can you change to something to this effect?
`Represents the properties backing AEP UI that can be changed on the interaction with the UI.`